src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.ts

```diff
@@ -47,7 +47,7 @@ export class RetrospectTaskModalComponent implements OnDestroy, AfterViewChecked
     overlayNoRowsTemplate = '<span>No Members for this Issue!</span>';
     expandedDescHidden = true;
     allowDescViewToggle = true;
-    descMaxHeight = 90;
+    descMaxHeight = 40;
```

Comment:
Please update the comment in the`ngAfterViewChecked` method accordingly. Also, we should set `allowDescViewToggle` to true, in the `ngAfterViewChecked` method, if the visible length is greater than descMaxHeight.

Reply:
done.